docs/sources/dashboards/variables/_index.md

```diff
@@ -15,57 +15,78 @@ weight: 800
 
-Variables allow you to create more interactive and dynamic dashboards. Instead of hard-coding things like server, application,
-and sensor names in your metric queries, you can use variables in their place. Variables are displayed as dropdown lists at the top of
-the dashboard. These dropdowns make it easy to change the data being displayed in your dashboard.
+Variables allow you to create more interactive dashboards.
+Instead of hard-coding things like server, application, and sensor names in your metric queries, you can use variables in their place.
+They're useful for administrators who want to allow Grafana viewers to adjust visualizations without giving them full editing permissions.
+
+Using variables also allows you to single-source dashboards.
+If you have multiple identical data sources or servers, you can make one dashboard and use variables to change what you are viewing.
+This simplifies maintenance and upkeep enormously.
+
+{{< youtube id="mMUJ3iwIYwc" >}}
 
-{{< figure src="/static/img/docs/v50/variables_dashboard.png" alt="Variable drop-down open and two values selected" >}}
+You can use variables in:
 
-{{< docs/play title="Templating - Global variables and interpolation" url="https://play.grafana.org/d/HYaGDGIMk/" >}}
+- Data source queries
+- [Panel repeating options](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/configure-panel-options/#configure-repeating-panels)
+- [Dashboard and panel links](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/build-dashboards/manage-dashboard-links/)
+- Titles
+- Descriptions
+- [Transformations](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/)
 
-Variables are useful for administrators who want to allow Grafana viewers to adjust visualizations without giving them full editing permissions. Grafana viewers can use variables.
+To see variable settings, navigate to **Dashboard Settings > Variables**.
+Click a variable in the list to see its settings.
 
-Variables and templates also allow you to single-source dashboards. If you have multiple identical data sources or servers, you can make one dashboard and use variables to change what you are viewing. This simplifies maintenance and upkeep enormously.
+{{< docs/play title="Templating - Interactive dashboard" url="https://play.grafana.org/goto/B9Xog68Hg?orgId=1" >}}
 
-## Templates
+## Template variables {#templates}
 
 A _template_ is any query that contains a variable.
+Queries with text that starts with `$` are templates.
 
-For example, if you were administering a dashboard to monitor several servers, you _could_ make a dashboard for each server. Or you could create one dashboard and use panels with template queries like this one:
+For example, if you were administering a dashboard to monitor several servers, you _could_ make a dashboard for each server.
```

Comment:
Just noting that this servers example phrasing is very much alike the one in the beginning, I just wanted to bring awareness

Reply:
Good call out. I updated it so it wasn't so repetitive and added a screenshot.